apps/src/templates/projects/projectsRedux.js

```diff
@@ -145,6 +148,32 @@ function personalProjectsList(state = initialPersonalProjectsList, action) {
         ...state,
         projects: projects,
       };
+    case START_RENAMING_PROJECT:
+      var projectBeingRenamed = action.projectId;
+
+      var projectBeingRenamedIndex = state.projects.findIndex(project => project.channel = projectBeingRenamed);
+
+      var updatedEditing = [...state.projects];
+      updatedEditing[projectBeingRenamedIndex].isEditing = true;
+
+      updatedEditing[projectBeingRenamedIndex].updatedName = action.updatedName;
```

Comment:
Here, you're replacing the `projects` array, but you're mutating the actual project objects within it, which ideally a reducer will never do.  I think what you want is:
```js
updatedEditing[projectBeingRenamedIndex] = {
  ...updatedEditing[projectBeingRenamedIndex],
  isEditing: true,
  updatedName: action.updatedName,
};
```